deeposlandia/webapp/templates/predictor.html

```diff
@@ -132,6 +132,20 @@ <h2>Making of</h2>
   </ul>
 </p>
 
+<div class=infoframe>
+  <h4>GPDR compliance</h4>
+  <p>
+    The images are stored on our server when the uploading action is done. We
+    do not use them in any other purpose than this web
+    application. However we commit to delete the uploaded images on a
```

Comment:
Suggestion:

"And we suppress the uploaded images every two weeks."

Reply:
Done.